packages/components/src/components/segmented-control/segmented-control-item/segmented-control-item-styles.spec.ts

```diff
@@ -1,6 +1,6 @@
 import { getComponentCss } from './segmented-control-item-styles';
 
-xdescribe('getComponentCss()', () => {
+describe('getComponentCss()', () => {
```

Comment:
Ah, because optional last 2 parameters (currently after theme) these tests are now incomplete.

Reply:
fixed it